src/open_inwoner/configurations/bootstrap/templates/digid_oidc.rst.template

```diff
@@ -0,0 +1,11 @@
+{% extends "base.rst.template" %}
+
+{% block link -%}
+.. _digid_oidc:
+{% endblock -%}
+
+{% block title -%}
+========================
+DigiD OIDC configuration
+========================
+{% endblock %}
```

Comment:
I feel like having a separate template for each config is not completely necessary? The only differing elements are the rst link and the name, which we could probably configure somewhere. This is not super crucial right now though